Sources/Internal/FrameProvider.swift

```diff
@@ -347,7 +412,13 @@ final class FrameProvider {
   // boundary month that's only showing a subset of days.
   private func numberOfWeekRows(in month: Month) -> Int {
     let rowOfLastDateInMonth: Int
-    if month == content.monthRange.lowerBound {
+    if month == content.monthRange.lowerBound && month == content.monthRange.upperBound {
+      let firstDayOfOnlyMonth = content.dayRange.lowerBound
+      let lastDayOfOnlyMonth = content.dayRange.upperBound
+      let rowOfFirstDayOfOnlyMonth = adjustedRowInMonth(for: firstDayOfOnlyMonth)
+      let rowOfLastDayOfOnlyMonth = adjustedRowInMonth(for: lastDayOfOnlyMonth)
+      rowOfLastDateInMonth = rowOfLastDayOfOnlyMonth - rowOfFirstDayOfOnlyMonth
```

Comment:
this new case fixes an issue where the calculation for the height of a month would be incorrect if it was both the first and last month in the calendar. This new case handles that properly.